lib/bootstrap.js

```diff
@@ -0,0 +1,65 @@
+var spawn = require('child_process').spawn;
+var path = require('path');
+var sleep = require('sleep').sleep;
+
+function Bootstrap(codecept, callback) {
+
+  var phantomjs, seleniumPid, callbackCalled = false;
+
+  (function () {
+
+    var protractor = codecept.config.helpers.Protractor;
```

Comment:
Yeah, sure!! I made this feature using only  `Protractor` because my app use `Angular`. I don't use `WebDriverIO`, but  I can change a refactory to check the others helpers.